fab/fabfile.py

```diff
@@ -360,27 +347,6 @@ def webworkers():
 
 @task
 @roles(ROLES_ALL_SRC)
-def install_packages():
-    """Install packages, given a list of package names"""
-    _require_target()
-
-    if what_os() == 'ubuntu':
-        packages_list = 'apt-packages.txt'
-        installer_command = 'apt-get install -y'
-    else:
-        return
-
-    packages_file = posixpath.join(PROJECT_ROOT, 'requirements', packages_list)
-
-    with open(packages_file) as f:
-        packages = f.readlines()
-
-    sudo("%s %s" % (installer_command,
-                    " ".join(map(lambda x: x.strip('\n\r'), packages))), user='root')
-
-
-@task
-@roles(ROLES_ALL_SRC)
```

Comment:
haha this diff confused me so much when i was reviewing it. git diff should weight contiguous lines of code more strongly